contentcuration/contentcuration/frontend/settings/pages/Storage/RequestForm.vue

```diff
@@ -507,4 +508,8 @@
     margin-bottom: 8px;
   }
 
```

Comment:
```suggestion
.license-link {
  font-size: 14px;
}
  
// fixes unintended margin caused by KDS styles
.license-link ::v-deep span {
  margin-left: 0 !important;
}
```

- To make the link font size match that of paragraphs
- Tracking comment for `::v-deep`

Reply:
@yeshwanth235 I don't see the change of `font-size` applied - would you double-check?